version/distro/distro.go

```diff
@@ -31,6 +31,7 @@ const (
 	WDMyCloud = Distro("wdmycloud")
 	Unraid    = Distro("unraid")
 	Alpine    = Distro("alpine")
+	IStoreOS  = Distro("istoreos")
```

Comment:
Careful here. These don't yet represent a hierarchy. We've had two aborted attempts at this already which is why Ubuntu isn't in this list. By adding this you remove any OpenWRT specialization elsewhere that checks for OpenWRT.

What's the goal with this?

Reply:
iStoreOS is a fairly popular NAS in China. Right now we can count these systems by hostname, but we're undercounting anyone who changed from the default "iStoreOS"